apps/src/gamelab/AnimationTab/PiskelEditor.jsx

```diff
@@ -65,6 +70,39 @@ const PiskelEditor = React.createClass({
     if (newProps.selectedAnimation !== this.props.selectedAnimation) {
       this.loadSelectedAnimation_(newProps);
     }
+    if (newProps.pendingAnimationAddition
+      && newProps.selectedAnimation === newProps.pendingAnimationAddition.key) {
```

Comment:
Do we have to compare `newProps.pendingAnimationAddition` with `this.props.pendingAnimationAddition` or do we end up clearing this out after we've added it?